Include/CollidableModel.h

```diff
@@ -1,13 +1,29 @@
 #pragma once
+#define GLM_ENABLE_EXPERIMENTAL
 #include "Model.h"
+#include "Parameters.h"
 #include <GL/glew.h>
 #include <GLFW/glfw3.h>
 #include <glm/glm.hpp>
 #include <glm/gtc/matrix_transform.hpp>
 #include <glm/common.hpp>
 #include <glm/gtc/type_ptr.hpp>
+#include <glm/gtx/matrix_decompose.hpp>
 
 class CollidableModel : public Model {
 public:
-	CollidableModel(GLuint VAO, GLuint numVertices, glm::vec3 color);
+	CollidableModel(GLuint VAO, GLuint colliderVAO, GLuint numVertices, glm::vec3 color);
```

Comment:
Why do we pass the colliderVAO as well? Can't CollidableModel pick one from the Assets singleton itself, since it shouldn't be different than the cube?